beakerx/js/src/shared/bkUtils.js

```diff
@@ -58,6 +58,9 @@ define([
     },
     newDeferred: function() {
       return jQuery.Deferred();
+    },
+    sanitizeRegExpString: function(string) {
```

Comment:
actually prefer to allow user to type a regexp.
just change the hint text in the box from "search..." to "search regexp ..."